packages/gill/src/__typetests__/simulate-transaction.ts

```diff
@@ -10,8 +17,11 @@ const rpcTestnet = null as unknown as RpcTestnet<SimulateTransactionApi>;
 const rpcMainnet = null as unknown as RpcMainnet<SimulateTransactionApi>;
 
 const baseTransaction = null as unknown as Transaction;
-const compilableTransaction = null as unknown as CompilableTransactionMessage;
-const signedTransaction = null as unknown as FullySignedTransaction;
+const compilableTransaction = null as unknown as (TransactionMessage & TransactionMessageWithFeePayer);
+const signedTransaction = null as unknown as (FullySignedTransaction & Readonly<{
+  messageBytes: TransactionMessageBytes;
+  signatures: SignaturesMap;
+}>);
```

Comment:
You should be able to do `null as unknown as Transaction & FullySignedTransaction` here